pkg/features/list.go

```diff
@@ -126,4 +126,7 @@ var (
 
 	// Display RHSA/RHBA/RHEA advisory separately from associated CVE.
 	CVEAdvisorySeparation = registerFeature("Display RHSA/RHBA/RHEA advisory separately from associated CVE", "ROX_CVE_ADVISORY_SEPARATION")
+
+	// Add the ability to generate an SBOM from an image
+	SBOMGeneration = registerFeature("Add the ability to generate an SBOM from an image", "ROX_SBOM_GENERATION")
```

Comment:
nit: Thoughts on enabling by default?

```suggestion
	SBOMGeneration = registerFeature("Add the ability to generate an SBOM from an image", "ROX_SBOM_GENERATION", enabled)
```

Reply:
Off by default allows UI team to build feature even before Central has finished the API.